daml-lf/spec/daml-lf-1.rst

```diff
@@ -3662,40 +3665,152 @@ Int64 functions
   in ``Some``. If the input does not match the regexp ``[+-]?\d+`` or
   if the result of the conversion overflows, returns ``None``.
 
+MathContext functions
+~~~~~~~~~~~~~~~~~~~~~
+
+*  ``'MATHCTX_UNLIMITED' : MathContext``
+
+   Returns `'UnBondedMathContext'`
+
+* ``MATHCTX_BOUNDED' : RoundingMode -> Int64 -> MathContext``
+
+   Returns ``'BoundedMathContext' v₁ v₂`` where ``v₁`` and ``v₂`` respectively the first and second argument.
```

Comment:
I think a function  ``NUMERIC_MATHCTX_BOUNDED' : ∀ (α : nat). RoundingMod -> MathContext``  can be Handy if we want to write library for numeric.

Reply:
Why do we need that in daml-lf instead of keeping it as a pure Daml thing?